src/pages/index.js

```diff
@@ -1,13 +1,25 @@
 import React from "react"
 import SEO from "../components/SEO"
-import Hero from "../components/Hero/hero"
-import ArticlesSection from "../components/Articles/articles-section"
+import Hero from "../molecules/Hero"
+import FeaturedSection from "../atoms/FeaturedSection"
 
 const HomePage = () => (
   <>
     <SEO />
     <Hero />
-    <ArticlesSection />
+    <FeaturedSection level="1" title="Para quem é o Educodar?">
+      <p>
+        Se você tem pelo menos 14 anos e estuda em uma escola pública de Belo
+        Horizonte (MG) ou Porto Alegre (RS), você pode participar como estudante
```

Comment:
For text flow control & accessibility, put city names with non-breaking space and add abbreviations:
```jsx
Belo&nbsp;Horizonte&nbsp;(<abbr title="Minas Gerais">MG</abbr>) <wbr />
ou Porto&nbsp;Alegre&nbsp;(<abbr title="Rio Grande do Sul">RS</abbr>)
```

Reply:
Done.